OpenQA/Isotovideo/CommandHandler.pm

```diff
@@ -30,6 +33,8 @@ has [qw(current_test_name current_test_full_name)];
 # the currently processed test API function
 has current_api_function => undef;
 
+has status => 'initial';
```

Comment:
Maybe add a comment (similarly to the other variables) that this variable holds the overall test status and is either `'initial'`, `'running'` or `'finished'`.

Reply:
done